llama_stack/distribution/routing_tables/vector_dbs.py

```diff
@@ -74,3 +87,135 @@ async def unregister_vector_db(self, vector_db_id: str) -> None:
         if existing_vector_db is None:
             raise ValueError(f"Vector DB {vector_db_id} not found")
         await self.unregister_object(existing_vector_db)
+
+    async def openai_retrieve_vector_store(
+        self,
+        vector_store_id: str,
+    ) -> VectorStoreObject:
+        await self.get_vector_db(vector_store_id)
+        return await self.get_provider_impl(vector_store_id).openai_retrieve_vector_store(vector_store_id)
+
+    async def openai_update_vector_store(
+        self,
+        vector_store_id: str,
+        name: str | None = None,
+        expires_after: dict[str, Any] | None = None,
+        metadata: dict[str, Any] | None = None,
+    ) -> VectorStoreObject:
+        await self.get_vector_db(vector_store_id)
```

Comment:
what does this do? you have a "get" method call and you aren't assigning the return value. what is the intent?

Reply:
I used read permissions to trigger the ACL but that was wrong. I updated things to actually do the access control. Sorry for the noise there.